apps/site/assets/css/_checkbox.scss

```diff
@@ -3,7 +3,9 @@
   position: relative;
 }
 
+
 .c-checkbox__input {
+  /* stylelint-disable-next-line property-no-vendor-prefix */
```

Comment:
As in `_radio.scss`, can we instead use `appearance: none` and let autoprefixer handle it? And in that case, would we still need `opacity: 0` or `width: 0`?

Reply:
Given the comments, this looked like a case where we _only_ wanted this for FF, and the opacity setting serves a certain purpose. I was hesitant to modify this, mostly because of the testing effort involved to ensure it works as intended however many years/months ago when this was written.